### ISSUE TITLE
Do not run tests decorated with WONTFIX resolution BZs

### ISSUE THREAD

**Issue by rochacbruno** [SatelliteQE/robottelo#4261]
Once #4260 is done

- [x] Use `pytest_collection_modifyitems` hook to filter out all the WONTFIX decorated tests from the test collection and ensure the information is logged `INFO: de-selecting N wontfix from tests`
